requirements-dev.txt

```diff
@@ -34,3 +34,5 @@ jsmin
 # for Silicon Labs Configurator (SLC)
 websockets
 colorama
+
+huffman
```

Comment:
This is our own version, in a submodule. It is in `.gitmodules`, so it shouldn't be a PyPi requirement.
```
[submodule "tools/huffman"]
	path = tools/huffman
	url = https://github.com/tannewt/huffman.git
```

Reply:
Absolutely, changing the make target solves all the problems, I didn't realize it because I hadn't corrected it in that order.